src/static/partials/manage-ncs.html

```diff
@@ -22,6 +23,7 @@
           <td>{{ coordinator.email }}</td>
           <td>{{ coordinator.phone }}</td>
           <td id="number">{{ coordinator.created }}</td>
+	  <td><span>{{coordinator.user_status}}</span></td>
```

Comment:
add **span** tag to the remaing **td** elements as well so that code will be remain uniform at all places.